Source/dom/import.js

```diff
@@ -0,0 +1,35 @@
+import { wrapNativeObject } from './wrapNativeObject'
+
+function _importWithImporter(data, importer) {
+  importer.prepareToImportFromData(data)
+  return wrapNativeObject(importer.importAsLayer())
+}
+
+function _importSVG(data) {
+  return _importWithImporter(data, MSSVGImporter.svgImporter())
+}
+function _importPDF(data) {
+  return _importWithImporter(data, MSPDFImporter.pdfImporter())
+}
+function _importEPS(data) {
+  return _importWithImporter(data, MSPDFImporter.epsImporter())
+}
+
+export const layerFromData = (data, type) => {
+  switch (type) {
+    case 'svg':
+      return _importSVG(data)
+    case 'pdf':
+      return _importPDF(data)
+    case 'eps':
+      return _importEPS(data)
+    default:
+      throw new Error(`Can't import ${type}`)
+  }
+}
+
+export const layerFromString = (str, type) => {
```

Comment:
I don't think we need this tbh

Reply:
Removed. This was just to make it easy to create from SVG strings but it seems that can happen with a Buffer now.